tests/monster_test_generated.rs

```diff
@@ -1179,7 +1179,7 @@ impl<'a> Monster<'a> {
   pub fn testnestedflatbuffer(&self) -> Option<&'a [u8]> {
     self._tab.get::<flatbuffers::ForwardsUOffset<flatbuffers::Vector<'a, u8>>>(Monster::VT_TESTNESTEDFLATBUFFER, None).map(|v| v.safe_slice())
   }
-  pub fn testnestedflatbuffer_nested_flatbuffer(&'a self) ->  Option<Monster<'a>> {
+  pub fn testnestedflatbuffer_nested_flatbuffer(&'a self) -> Option<Monster<'a>>{
```

Comment:
Why did this change?

Reply:
I thought double spaces look bad. But, removed space after the return type is my mistake. I fixed removed space.